pkg/cmd/adm/register_member.go

```diff
@@ -71,10 +71,7 @@ func NewRegisterMemberCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			term := ioutils.NewTerminal(cmd.InOrStdin, cmd.OutOrStdout)
 			ctx := newExtendedCommandContext(term, client.DefaultNewClientFromRestConfig)
-			newCommand := func(name string, args ...string) *exec.Cmd {
-				return exec.Command(name, args...)
-			}
-			return registerMemberCluster(ctx, newCommand, 5*time.Minute, commandArgs)
+			return registerMemberCluster(ctx, 5*time.Minute, commandArgs)
```

Comment:
Maybe worth adding a new arg for the timeout? Our clusters are sometimes rather busy...

Reply:
Good idea done in https://github.com/kubesaw/ksctl/pull/53/commits/7103ce60fde7119673797d7fa70c29ab3202d8e0

I like the idea of exposing this to the end user  (with some default value ), even if I hope there will be no need for them to increase this timeout further.